image_data_guide/04c_pytorch_training.ipynb

```diff
@@ -0,0 +1,1126 @@
+      "\u001b[34m[2020-10-09 18:19:58.172 algo-1:63 WARNING hook.py:921] var is not Tensor or list or tuple of Tensors, module_name:RandomHorizontalFlip Image\u001b[0m\n",
+      "\u001b[34mtrain Loss: 2.1853 Acc: 0.2532\u001b[0m\n",
+      "\u001b[34mval Loss: 1.6451 Acc: 0.4800\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 1/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\u001b[34mtrain Loss: 1.5451 Acc: 0.5082\u001b[0m\n",
+      "\u001b[34mval Loss: 1.1039 Acc: 0.6436\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 2/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\u001b[34mtrain Loss: 1.2770 Acc: 0.5714\u001b[0m\n",
+      "\u001b[34mval Loss: 0.8742 Acc: 0.7273\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 3/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\u001b[34mtrain Loss: 1.1721 Acc: 0.6209\u001b[0m\n",
+      "\u001b[34mval Loss: 0.8072 Acc: 0.7418\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 4/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\u001b[34mtrain Loss: 1.0837 Acc: 0.6500\u001b[0m\n",
+      "\u001b[34mval Loss: 0.7410 Acc: 0.7600\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 5/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\u001b[34mtrain Loss: 1.0157 Acc: 0.6568\u001b[0m\n",
+      "\u001b[34mval Loss: 0.6945 Acc: 0.7709\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 6/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\u001b[34mtrain Loss: 0.9779 Acc: 0.6732\u001b[0m\n",
+      "\u001b[34mval Loss: 0.6952 Acc: 0.7636\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 7/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\u001b[34mtrain Loss: 0.9395 Acc: 0.6850\u001b[0m\n",
+      "\u001b[34mval Loss: 0.6718 Acc: 0.7673\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 8/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\u001b[34mtrain Loss: 0.9341 Acc: 0.6905\u001b[0m\n",
+      "\u001b[34mval Loss: 0.6600 Acc: 0.7782\n",
+      "\u001b[0m\n",
+      "\u001b[34mEpoch 9/9\u001b[0m\n",
+      "\u001b[34m----------\u001b[0m\n",
+      "\n",
+      "2020-10-09 18:21:04 Uploading - Uploading generated training model\u001b[34mtrain Loss: 0.9324 Acc: 0.6991\u001b[0m\n",
+      "\u001b[34mval Loss: 0.6519 Acc: 0.7927\n",
+      "\u001b[0m\n",
+      "\u001b[34mTraining complete in 1m 10s\u001b[0m\n",
+      "\u001b[34mBest val Acc: 0.792727\u001b[0m\n",
+      "\u001b[34m2020-10-09 18:20:59,684 sagemaker-training-toolkit INFO     Reporting training SUCCESS\u001b[0m\n",
+      "\n",
+      "2020-10-09 18:21:17 Completed - Training job completed\n",
+      "Training seconds: 210\n",
+      "Billable seconds: 210\n"
+     ]
+    }
+   ],
+   "source": [
+    "pytorch_estimator.fit(data_channels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<pre>\n",
+    "</pre>"
+   ]
+  },
+  {
```

Comment:
Somewhere here add a quick blurb describing what the integers in the category_labels dictionary mean and/or where they came from. e.g. "Recall that in a previous notebook you assigned each class an integer. This is because computers don't know what a 'cat' or 'dog' is...."

Reply:
added note about how the indexing data formed in the download data notebook